monitoring/grafana/dashboards/tox.ini

```diff
@@ -20,3 +20,14 @@ passenv = GRAFONNET_PATH
 commands =
      check: bash test-jsonnet.sh
      fix: jsonnet -m . jsonnet/grafana_dashboards.jsonnet
+
+[testenv:grafonnet-{query_test}]
+description =
+     Run promtool unit testing on grafana queries.
+deps =
+     -rtests/requirements.txt 
+setenv =
+commands =
+    behave tests/features --capture-stderr
+    mypy tests
+    isort tests
```

Comment:
I don't know exactly how to do this, but we need to ensure that these toxenvs run in a given order (grafana check and the test). Otherwise, since we are testing against JSON and not jsonnet, if someone changed jsonnet but didn't update the JSON files, the test might pass (the OLD JSONs are OK, but not the new ones)...

[This](https://tox.wiki/en/latest/config.html#conf-depends) might work but I remember trying that out a while ago and had some issues with it.

Reply:
done